src/proxy/ClientTable.php

```diff
@@ -177,6 +178,12 @@ protected function prepare()
      */
     protected function cleanup($sqlMode)
     {
+        try {
+            Yii::$app->db->createCommand('SELECT CONNECTION_ID()')->execute();
```

Comment:
@boehsermoe is this command safe for other dbms systems like:
- MySQL
- MariaDB
- SQLite
- PostgreSQL
- CUBRID
- Oracle
- MSSQL

Reply:
If not, maybe we should make another DAO valid task to check whether connection works?

Reply:
@boehsermoe maybe we could also add this command into the schema object check?

```
if ($this->db->schema instanceof \yii\db\mysql\Schema) {
```

Reply:
CONNECTION_ID only available in MySQL and MariaDB. It looks difficulty to handle all other systems.